### PR TITLE
zypper: add link and example about repos management

### PR DESCRIPTION
**Pull request by victorhck**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/zypper.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # zypper
 
 > SUSE & openSUSE package management utility.
+> More information: <https://en.opensuse.org/SDB:Zypper_manual>.
 
 - Synchronize list of packages and versions available:
```
Comment (navarroaxel) [inline pages/linux/zypper.md:6]: @victorhck do you want to update this example description?

File: pages/linux/zypper.md [hunk 2]
```diff
@@ -21,3 +22,7 @@
 - Search package via keyword:
 
 `zypper search {{keyword}}`
+
+- Show a list with information related with configured repositories:
```
Comment (marchersimon) [inline pages/linux/zypper.md:26]:
```suggestion
- Show information related to configured repositories:
```
Would this also be correct? (`list of information` = `information` in my opinion)

File: pages/linux/zypper.md (hunk 1 above)
Comment (victorhck) [reply pages/linux/zypper.md:6]: > @victorhck do you want to update this example description?

Wich one?

File: pages/linux/zypper.md (hunk 2 above)
Comment (victorhck) [reply pages/linux/zypper.md:26]: yep! Maybe better in that way!

File: pages/linux/zypper.md (hunk 1 above)
Comment (marchersimon) [reply pages/linux/zypper.md:6]: Line 6

Comment (victorhck) [reply pages/linux/zypper.md:6]: I'm a openSUSE user, but not a English speaker, so please fix if needed! :)
Maybe the current text can be replaced with:

> Download and refresh the metadata of repository list

**Review by schneiderl — APPROVED** [review]

File: pages/linux/zypper.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/zypper.md:6]: If I check the pages `apt` and `pacman-sync` the current example description is fine for us, but we can hear other opinions.

Comment (sbrl) [reply pages/linux/zypper.md:6]: I'm a bit confused. What about the existing description needs changing?

Comment (victorhck) [reply pages/linux/zypper.md:6]: > If I check the pages `apt` and `pacman-sync` the current example description is fine for us, but we can hear other opinions.

> I'm a bit confused. What about the existing description needs changing?

So, let's left the text as is. Ok for me too...

File: pages/linux/zypper.md [hunk 3]
```diff
@@ -21,3 +22,7 @@
 - Search package via keyword:
 
 `zypper search {{keyword}}`
+
+- Show information related to configured repositories:
+
+`zypper repos -P`
```
Comment (navarroaxel) [inline pages/linux/zypper.md:28]: We prefer long argument format when is possible:
```suggestion
`zypper repos --sort-by-priority`
```
Why do you think `--sort-by-priority` is an essential flag for this command?

**Comment by navarroaxel** [comment]
@victorhck remember that you can add new pages :hugs: specific for subcommands like `zypper repos`, `zypper update`, `zypper list-updates`, etc like was done with `git` or `pacman` (the packager for Arch Linux).

File: pages/linux/zypper.md (hunk 3 above)
Comment (victorhck) [reply pages/linux/zypper.md:28]: When you run 
> zypper repos

zypper suggest to run zypper repos -P (or the long argument) to get more information. Just add a column with priority of each repo. Sometime it's useful to know the repo's priority

Comment (navarroaxel) [reply pages/linux/zypper.md:28]: Ok, apply the long version and we can merge this.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]

**Comment by sbrl** [comment]
Thanks, @victorhck :-)

**Pull request merged**